enterprise_gateway/_version.py

```diff
@@ -3,4 +3,4 @@
 # Copyright (c) Jupyter Development Team.
 # Distributed under the terms of the Modified BSD License.
 
-__version__ = '2.1.1.dev4'
+__version__ = '2.1.1.dev5'
```

Comment:
We are building on top of EG 2.1.1
Can we change the version to 2.1.2dev5?

Reply:
Then it will be 2.1.2.dev1 right :P